man/systemd.exec.xml

```diff
@@ -89,6 +89,17 @@
 
     <variablelist class='unit-directives'>
 
+      <varlistentry>
+        <term><varname>BinaryDirectory=</varname></term>
+
+        <listitem><para>Takes a directory path relative to which the executable used by
+        the Exec*= (e.g. ExecStart=, ExecStop=, etc.) properties can be specified.
+        The name of the executable has to be non standard.
+        At the moment BinaryDirectory= will not take precedence over the default paths set at compilation time.
```

Comment:
```suggestion
        At the moment <varname>BinaryDirectory=</varname> will not take precedence over the default paths set at compilation time.
```
Changing this later, would be a breaking change, no? So maybe the `At the moment` should go as well.